src/platform/iotjs_module_i2c-linux-general.inl.h

```diff
@@ -134,6 +136,16 @@ int I2cSmbusAccess(int fd, uint8_t read_write, uint8_t command, int size,
 }
 
 
+int I2cSmbusReadByte(int fd) {
+  I2cSmbusData data;
+  int result;
+
+  result = I2cSmbusAccess(fd, I2C_SMBUS_READ, I2C_NOCMD, I2C_SMBUS_BYTE, &data);
+
+  return result >= 0 ? 0x0FF & data.byte : -1;
```

Comment:
@LaszloLango It is just bit masking for data.byte.
Does it need to make value '0x0FF' to variable?